general/ai/cli.go

```diff
@@ -59,25 +64,75 @@ func HowCmd(c *cli.Context) error {
 			return err
 		}
 		log.Output(coreutils.PrintLink(llmAnswer))
+
+		err = sendFeedback()
+		if err != nil {
+			return err
+		}
+
 		log.Output("\n" + coreutils.PrintComment("-------------------") + "\n")
 	}
 }
 
+type QuestionBody struct {
+	Question string `json:"question"`
```

Comment:
Let's check if `Question` can be made private.

Reply:
if it will be private it won't be sent to the server @eyalbe4